src/main/java/App.java

```diff
@@ -6,6 +6,7 @@ public String getGreeting() {
         return "Hello world.";
     }
 
+    public String redundantVfar;
     public static void main(String[] args) {
         System.out.println(new App().getGreeting());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)